roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -188,7 +188,7 @@
     ("credentials", _("{{ horizon.websso.choices.credentials }}")),
     {% if keystone.federation.sp.oidc.enabled|bool %}
     {% for sp in keystone.federation.sp.oidc.providers_info %}
-    ("{{ sp.name }}-OIDC-IDP", _("{{ sp.horizon_name }}")),
+    ("{{ sp.idp_name }}-OIDC-IDP", _("{{ sp.horizon_name }}")),
```

Comment:
This seems like an unrelated change. 

Where are we tracking what the supported data structure of `keystone.federation.sp.oidc.providers_info` ?